moped-database/migrations/1697055656000_component_agol_view/up.sql

```diff
@@ -0,0 +1,174 @@
+DROP VIEW IF EXISTS component_arcgis_online_view;
+CREATE OR REPLACE VIEW component_arcgis_online_view AS (
+    SELECT
+        mpc.project_id,
+        comp_geography.project_component_id,
+        comp_geography.feature_ids,
+        mpc.component_id,
+        comp_geography.geometry,
+        comp_geography.signal_ids,
+        council_districts.council_districts,
+        comp_geography.length_feet_total,
+        mc.component_name AS component_name,
+        mc.component_subtype AS component_subtype,
+        mc.component_name || ' - ' || mc.component_subtype AS component_name_full,
+        subcomponents.subcomponents,
+        work_types.work_types,
+        component_tags.component_tags,
+        mpc.description AS component_description,
+        mpc.is_deleted is_project_component_deleted,
+        plv.is_deleted is_project_deleted,
+        mpc.interim_project_component_id,
+        mpc.completion_date,
+        mpc.srts_id,
+        mpc.location_description,
+        plv.project_name,
+        plv.project_description,
+        plv.ecapris_subproject_id,
+        plv.updated_at,
+        mpc.phase_id AS component_phase_id,
+        mph.phase_name AS component_phase_name,
+        mph.phase_name_simple as component_phase_name_simple,
+        current_phase.phase_id AS project_phase_id,
+        current_phase.phase_name AS project_phase_name,
+        current_phase.phase_name_simple AS project_phase_name_simple,
+        COALESCE(mph.phase_name, current_phase.phase_name) AS current_phase_name,
+        COALESCE(mph.phase_name_simple, current_phase.phase_name_simple) AS current_phase_name_simple,
+        plv.project_team_members,
+        plv.project_sponsor,
+        plv.project_lead,
+        plv.public_process_status,
+        plv.interim_project_id,
+        plv.project_partner,
+        plv.task_order_name,
+        plv.funding_source_name,
+        plv.type_name,
+        plv.project_note,
+        '2000-01-01T00:00:00' as project_note_date,
```

Comment:
just out of curiosity, is this a placeholder for `date_created`?

Reply:
just started reviewing https://github.com/cityofaustin/atd-moped/pull/1162 so this makes more sense now! didn't realize it was in the project list view DB view yet 👍

Reply:
yep!!!